plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/BaseRaptorConnectorTest.java

```diff
@@ -102,6 +102,12 @@ protected TestTable createTableWithDefaultColumns()
         throw new SkipException("Raptor connector does not support column default values");
     }
 
+    @Override
+    protected void verifyConcurrentInsertFailurePermissible(Exception e)
```

Comment:
This needs to be `verifyConcurrentUpdateFailurePermissible`

Reply:
Changed.